docs/index.rst

```diff
@@ -735,20 +735,28 @@ Network
   - **speed**: the NIC speed expressed in mega bits (MB), if it can't be
     determined (e.g. 'localhost') it will be set to ``0``.
   - **mtu**: NIC's maximum transmission unit expressed in bytes.
+  - **flags**: a string of comma-separated flags on the interface (may be the empty string).
```

Comment:
"may be AN empty string"

Reply:
Fixed in d83c959a981c68ff7ff3d60a155b4f35640e9fa9.